tokens/media.yml

```diff
@@ -0,0 +1,54 @@
+media:
+  $extensions:
+    com.redhat.ux:
+      use: breakpoint
+      heading: Media Queries
+      description: |
+        For use in templating systems and JavaScript modules. We recommend against importing these
+        tokens directly into components. These tokens are directly derived from
+        [breakpoints](#breakpoints).
+
+        ```html
+        <link rel="stylesheet"
+              href="large-screen.css"
+              media="%(%%(% tokens.get('--rh-media-desktop-large') %)%%)%"
+        ```
+
+  mobile:
+    portrait:
+      $type: mediaQuery
+      value:
+        max-width: '{breakpoint.mobile.portrait}'
+    landscape:
+      $type: mediaQuery
+      value:
+        min-width: 576px
```

Comment:
It would be awesome to be able to javascript here so we could tokenize these values as well but I'm sure it's a lot of hoop jumping.  Something like:

```
   min-width: `addPixels('{breakpoint.mobile.portrait}', 1px)`
```

Reply:
yeah that's on style-dictionary's backlog, but expressions in references aren't available yet